axelor-base/src/main/java/com/axelor/apps/base/service/message/TemplateMessageServiceBaseImpl.java

```diff
@@ -130,6 +130,43 @@ public File generateBirtTemplate(
     }
   }
 
+  public String generateBirtFileLink(
+      TemplateMaker maker,
```

Comment:
Create a method for common code between generateBirtFileLink and generateBirtTemplate.